actions/install/installer.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# SLSA verifier install script
+
+verifier_release="$1"
+install_dir="$HOME/.slsa/bin/${verifier_release}"
+
+shopt -s expand_aliases
+if [ -z "$NO_COLOR" ]; then
+  alias log_info="echo -e \"\033[1;32mINFO\033[0m:\""
+  alias log_error="echo -e \"\033[1;31mERROR\033[0m:\""
+else
+  alias log_info="echo \"INFO:\""
+  alias log_error="echo \"ERROR:\""
+fi
+set -e
```

Comment:
put it at the top of the file. Also, `set -euo pipefail` is more comprehensive, I think

Reply:
Done